### DISCUSSION TITLE
Hello Everyone!

### DISCUSSION THREAD

**Discussion by eyehwan** [oss-compass/community#1] [General]
General discussion channel is to broadcast news, create conversation in your community, answer questions, and share ideas. 
Welcome!

**Comment by yongqiangyang** [comment]
请问可以更新一下微信交流群二维码嘛？希望加入讨论~

**Reply by nanzm** [reply]
@normal-coder

**Reply by eyehwan** [reply]
@yongqiangyang  Just updated: https://oss-compass.org/docs/zh/community/wechat/
